spec/lib/miq_expression/field_spec.rb

```diff
@@ -77,6 +77,16 @@
     end
   end
 
+  describe '#parse_report_field' do
+    it 'returns the correct format for a non field' do
+      expect(MiqExpression::Field.parse_report_field('Vm.managed-environment')).to eq('managed.environment')
```

Comment:
Ah! This is because it's a tag? I think you want `MiqExpression.parse_field_or_tag` for that :grin: